odl/discr/lp_discr.py

```diff
@@ -688,6 +697,26 @@ def show(self, title=None, method='', indices=None, show=False, fig=None,
 
         from odl.util.graphics import show_discrete_data
 
+        if coords is not None:
+            if indices is not None:
+                raise ValueError('Cannot provide both coords and indices')
+
+            interv = self.space.uspace.domain
+            shape = self.shape
+            indices = []
+            for begin, end, n, coord in zip(interv.begin, interv.end,
+                                            shape, coords):
+                if coord is None:
+                    indices += [slice(None)]
+                else:
+                    if not begin <= coord <= end:
+                        raise ValueError('Coord {} not in range {}'
+                                         ''.format(coord, [begin, end]))
+
+                    # Todo: handle end = begin
```

Comment:
Throw an error? I guess using `None` is the only way not to run into rounding errors.